lmdk/libraries/smart_amp_test/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
@RanderWang do we have instructions on building a loadable module in our documentation? If so, could you please point me to it? If not, can you please add it?

Also, once this PR is merged, will our CI automatically start testing the smartamp as a loadable module? or will that be a follow up?

Reply:
@ranj063 Once this PR is merged, our CI will automatically start testing it as a loadable module.

Reply:
we have a batch shell to build it, I will also push it.    To test it in CI ? we need to first disable built-in smart amp and then remove smart amp in toml file ( waiting for toml file rework ?)

Reply:
BTW this is demo in lmdk directory, not in our building system, so more work to do for CI

Reply:
@RanderWang  @ranj063  CI automated testing can be planned as a follow up. My previous understanding was wrong. We needs to update CI storage and deployment to support loadable modules.